media-gfx/gimp/files/gimp-2.10.30_fix_crash_on_large_memory_allocation.patch

```diff
@@ -0,0 +1,61 @@
+From 4f99f1fcfd892ead19831b5adcd38a99d71214b6 Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to the gentoo bug + upstream commit link at the top of the patch

Reply:
Done.

\+ local variable `prepend` is renamed to `prename` due to `EbuildReservedName` CI error.